packages/create-sitecore-jss/src/templates/nextjs-personalize/src/components/CdpIntegrationScript.tsx

```diff
@@ -60,6 +61,11 @@ const CdpIntegrationScript = ({
           __html: `
               var _boxeverq = _boxeverq || [];
 
+              var currentHostname = window.location.hostname.substring(
+                4, 
```

Comment:
It's a magic number, for me it's not clear what this part of the code does. Does it take hostname after protocol? But sometimes you can have `http` and `https`. Use RegExp to extract the hostname

Reply:
done